src/Twilio/Rest/Authy/V1/Service/Entity/FactorList.php

```diff
@@ -81,10 +90,10 @@ public function create(string $binding, string $friendlyName, string $factorType
      *                        efficient page size, i.e. min(limit, 1000)
      * @return Stream stream of results
      */
-    public function stream(int $limit = null, $pageSize = null): Stream {
+    public function stream(array $options = [], int $limit = null, $pageSize = null): Stream {
```

Comment:
Are the `read/page/stream` the only part of this change that's breaking? Will the generated code samples also be updated?

I think this warrants a migration guide listing which resources this affects and how to update the client code.

Reply:
What generated code samples?

Reply:
I can't find any examples specific to these changes in the Twilio docs, but the code generator is also used to generate code samples for every API. Let's follow-up offline.